sdcm/sct_config.py

```diff
@@ -182,6 +182,10 @@ class SCTConfiguration(dict):
              type=str,
              help="Url to the repo of scylla manager version to install for management tests"),
 
+        dict(name="scylla_mgmt_agent_repo", env="SCT_SCYLLA_MGMT_AGENT_REPO",
```

Comment:
why this split is needed ? for a specific case of a bug in the agent ? or on a regular base we should have sperated repos ?

Reply:
it is necessary, as we are not testing agent installed on other OS than Centos (this is the same machine we install Scylla).
The manager (server) is installed on a monitoring machine, where we are able to use Centos, Ubuntu or Debian (although Debian wasn't tested yet)